view/theme/frio/php/standard.php

```diff
@@ -30,18 +30,18 @@
 	<meta name="viewport" content="initial-scale=1.0">
 	<meta request="<?php echo htmlspecialchars($_REQUEST['pagename']) ?>">
 	<script type="text/javascript">var baseurl="<?php echo DI::baseUrl() ?>";</script>
-	<script type="text/javascript">var frio="<?php echo "view/theme/frio"; ?>";</script>
+	<script type="text/javascript">var frio="<?php echo 'view/theme/frio'; ?>";</script>
```

Comment:
This is one of the most useless usages of a PHP command here, isn't it? :-)

Some ```<script type="text/javascript">var frio="view/theme/frio";</script>``` should be okay, shouldn't it?

Reply:
Both are okay, it just is a convention to use single quotes over double. Since there were some ugly `\"` in this file I converted all the quotes to the style we ask people to use.

Reply:
Indeed, maybe it was dynamic at some point?

Reply:
Now it's dynamic again :D

Reply:
It's dynamic - but is it needed to be dynamic?

Reply:
The hard-coded value was already stored in a variable, removing the variable itself would have been more invasive.